### PR TITLE
Add Matplotlib visualizations

### PR DESCRIPTION
**Pull request by AndreasPB**

### CODE REVIEW

**Comment by AndreasPB** [comment]
Waiting for everyone to approve(when they have had it up and running locallly)

**Review by nullxDEADBEEF — CHANGES_REQUESTED** [review]
Would be awesome to have labels on the y-axis in our visualizations + export them to pdf :DD

**Comment by Pankai222** [comment]
:shipit:

**Comment by Pankai222** [comment]
Would be awesome to have labels on the y-axis in our visualizations + export them to pdf ;))

**Comment by Primdahl26** [comment]
Would be awesome to have labels on the y-axis in our visualizations + export them to pdf :ii

File: src/haircut_plots.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# %%
+# Married and not married split
+df_m = df.drop(df.loc[df["Married"] == "NM"].index)
+df_nm = df.drop(df.loc[df["Married"] == "M"].index)
+
+# Binned by age in 10 year intervals
+bins = pd.cut(df["Age"], list(range(0, 100, 10)))
+
+df_nm.groupby(bins)["Tip amount"].mean().plot.line()
+df_m.groupby(bins)["Tip amount"].mean().plot.line()
```
Comment (Primdahl26) [inline src/haircut_plots.py]:
```suggestion
df_m.groupby(bins)["Tip amount"].mean().plot.line().set_ylabel('Tipping amount')
```

File: src/haircut_plots.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+plt.legend(["Not married", "Married"])
+
+# %% [markdown]
+# Exploring if culture has an impact on tip amounts
+
+# %%
+df_nm.groupby("Culture")["Tip amount"].mean().plot.bar()
+df_m.groupby("Culture")["Tip amount"].mean().plot.bar(color="orange")
```
Comment (Primdahl26) [inline src/haircut_plots.py]:
```suggestion
df_m.groupby("Culture")["Tip amount"].mean().plot.bar(color="orange").set_ylabel('Tipping amount')
```

**Review by Primdahl26 — DISMISSED** [review]
Could we also `poetry add notebook`?

**Comment by AndreasPB** [comment]
- Dashed/dotted til overlappende grafer
- Skalering af y-værdier(begge skal kunne bruge hele y-aksen(tip/age))
- Få styr på om bar graferne viser hvad du tror

File: src/haircut_plots.py (hunk 1 above)
Comment (AndreasPB) [reply src/haircut_plots.py]: Done

File: src/haircut_plots.py (hunk 2 above)
Comment (AndreasPB) [reply src/haircut_plots.py]: Done

**Comment by AndreasPB** [comment]
Så skulle det gerne være klar! :see_no_evil:

**Review by nullxDEADBEEF — COMMENTED** [review]
LGTM

**Review by nullxDEADBEEF — APPROVED** [review]
:shipit:

**Pull request merged**